### DISCUSSION TITLE
Add possibility to download asset without zipping

### DISCUSSION THREAD

**Discussion by JoeriE** [duncanmcclean/simple-commerce#997] [Feature Requests]
### Description

Hi,

Right now if you download the asset from the e-mail, the asset is archived in a .zip. It would be great if the asset if immediately downloaded. For example: if it's a zip, it is way harder to open / unarchive this on a phone. It would be more user-friendly if it immediately downloads the pdf.

Thanks!

**Comment by duncanmcclean** [comment]
This will be possible in the next version of Simple Commerce, which will be [released in the next month or so](https://github.com/duncanmcclean/simple-commerce?tab=readme-ov-file#upcoming-release).
